Install cleanup jeos #8363 ke (#678)
SUSE/doc-susemanager: modules/installation/pages/install-vm.adoc

// - 2019-06-19.

Enter the following settings when creating a new virtual machine using *virt-manager*.

[NOTE]
====

> In the following table minimal requirements are specified, which are suitable for a quick test installation (server with one client).

Reviewer: ```suggestion
This table specifies the minimum requirements.
These are suitable for a quick test installation, such as a server with one client.
```

Author: Thanks!  So much better

> If you want to use it as a production environment, consider the requirements listed in xref:hardware-requirements.adoc[].

Reviewer: ```suggestion
If you want to use a production environment, review the requirements listed in xref:hardware-requirements.adoc[].
```

. Create a new virtual machine using the downloaded JeOS KVM image and select [guimenu]``Import existing disk image``.
. Configure RAM and number of CPUs (at least 4 GB RAM and 2 CPUs).
. Name your KVM machine and select the [guimenu]``Customize configuration before install`` check box.

> . Click btn:[Add Hardware] and create three new virtual disks with the following specifications.

Reviewer: ```suggestion
. Click btn:[Add Hardware] to create three new virtual disks with these specifications.
```

| VirtIO Disk 4        | swap      | 4{nbsp}GB
|===

> . To boot the new VM from the JeOS image click btn:[Begin Installation].

Reviewer: ```suggestion
. Click btn:[Begin Installation] to boot the new VM from the JeOS image.
```
The original was better :stuck_out_tongue:

> Proceed through the basic JeOS installation prompts until the command prompt wait for input.

Reviewer: ```suggestion
Follow the prompts to complete the basic JeOS installation, until the process is complete and the command prompt waits for input.
```
Just trying to clarify this a bit.

[NOTE]
====

> Storage size values are the absolute minimum—suitable only for a small test or demo installation (server and one client).

Reviewer: ```suggestion
Storage size values are the absolute minimum.
They are suitable only for a small test or demonstration installation, such as a server with one client.
```
